src/uipath/_cli/_runtime/_contracts.py

```diff
@@ -832,12 +792,21 @@ def __init__(
         self.logs_exporter: Optional[Any] = None
         trace.set_tracer_provider(self.tracer_provider)
 
+    def add_span_processor(
```

Comment:
Can we also allow setting the trace_provider in the Runtime Factory ctor?
@akshaylive

Reply:
nvm